examples/pod_spec_path/README.md

```diff
@@ -0,0 +1,156 @@
+# Testing PodSpec path
```

Comment:
Hi @pedjak . Totally agree with having acceptance tests for this scenario. But this feature has been in demand by the community since April 23rd https://github.com/redhat-developer/service-binding-operator/issues/431 .Idk if a further delay in the feature point of view is reasonable. I guess we can take the acceptance test in a separate PR so that they(the community) gets to use this feature..This particular use case has been mentioned in every community call (from it's inception and till now)since April. Also, it provides with additional feedback once in master.

Reply:
PR from @pratikjagrut https://github.com/redhat-developer/service-binding-operator/pull/603